src/utils/maybeParseInt.ts

```diff
@@ -0,0 +1,7 @@
+export function maybeParseInt(str: string | null): number | null {
+  if (!str) {
+    return null;
+  }
+
+  return parseInt(str);
```

Comment:
The `parseInt` function may return `NaN` if the string does not contain a number. You should handle this case to ensure that the function returns `null` instead of `NaN`, to keep the return type consistent as `number | null`.